validate.py

```diff
@@ -4,6 +4,7 @@
 import requests
 import subprocess
 import time
+import datetime
```

Comment:
not used, though not important enough to wipe approvals over

Reply:
Oops, that was left in from the pre-time-refactor version.